data/json/monsters/mi-go.json

```diff
@@ -0,0 +1,201 @@
+[
+  {
+    "id": "mon_mi_go",
+    "type": "MONSTER",
+    "name": "mi-go",
+    "description": "This is an alien creature of uncertain origin.  Its shapeless pink body bears numerous sets of paired appendages of unknown function, and a pair of ribbed, membranous wings which seem to be quite useless.  Its odd, vaguely pyramid-shaped head bristles with numerous wavering antennae, and simply gazing upon the unnatural beast fills you with primordial dread.",
+    "default_faction": "mi-go",
+    "bodytype": "migo",
+    "species": [ "NETHER" ],
+    "volume": "92500 ml",
+    "weight": 120000,
+    "hp": 210,
+    "speed": 120,
+    "material": [ "flesh" ],
+    "symbol": "&",
+    "color": "pink",
+    "aggression": 20,
+    "morale": 30,
+    "melee_skill": 7,
+    "melee_dice": 4,
+    "melee_dice_sides": 6,
+    "melee_cut": 6,
+    "dodge": 4,
+    "armor_bash": 4,
+    "armor_cut": 12,
+    "vision_day": 50,
+    "harvest": "zombie_meatslug",
+    "path_settings": { "max_dist": 50 },
+    "special_attacks": [
+      [ "PARROT", 0 ],
```

Comment:
The cooldown is set to 0, does that mean that it will PARROT every turn or that it will never PARROT?

Reply:
That's the mi-go as it currently exists, so it'll parrot as often as the mi-go does. I haven't changed anything.